templates/archive-special.html

```
@@ -1,3 +1,113 @@
 <!-- wp:template-part {"slug":"header","area":"header"} /-->
- <!-- wp:post-content {"align":"wide","fontSize":"medium","layout":{"type":"default"}} /-->
-<!-- wp:template-part {"slug":"footer","area":"footer"} /-->
\ No newline at end of file
+
+<!-- wp:template-part {"slug":"breadcrumbs","area":"uncategorized"} /-->
+
+<!-- wp:cover {"useFeaturedImage":true,"dimRatio":50,"isUserOverlayColor":true,"customGradient":"linear-gradient(85deg,rgb(0,0,0) 0%,rgba(0,0,0,0) 100%)","contentPosition":"bottom center","isDark":false,"tagName":"section","metadata":{"name":"Hero"},"style":{"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"}}},"layout":{"type":"constrained"}} -->
+<section class="wp-block-cover is-light has-custom-content-position is-position-bottom-center" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40)"><span aria-hidden="true" class="wp-block-cover__background has-background-dim has-background-gradient" style="background:linear-gradient(85deg,rgb(0,0,0) 0%,rgba(0,0,0,0) 100%)"></span><div class="wp-block-cover__inner-container"><!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)